### PR TITLE
Second pass of renaming kube-dns configure files

### PR DESCRIPTION
**Pull request by MrHohn**
Continue work of #38523.

Not sure why cluster/centos/deployAddons.sh was omitted in previous PR. Also deletes the non-use `DNS_REPLICAS` var and changes `-rc` suffix in hack/local-up-cluster.sh.

@thockin @bowei @deads2k

### CODE REVIEW

**Comment by MrHohn** [comment]
@k8s-bot node e2e test this

File: docs/devel/running-locally.md [hunk 1]
```diff
@@ -159,7 +159,6 @@ To start the DNS service, you need to set the following variables:
 KUBE_ENABLE_CLUSTER_DNS=true
 KUBE_DNS_SERVER_IP="10.0.0.10"
 KUBE_DNS_DOMAIN="cluster.local"
-KUBE_DNS_REPLICAS=1
```
Comment (MrHohn) [inline docs/devel/running-locally.md]: Sure. Are we under the migration of this folder?

**Comment by thockin** [comment]
Exactly.  You see this PR is blocked because that file should not be edited

**Comment by MrHohn** [comment]
Sent kubernetes/community#174.

**Comment by MrHohn** [comment]
Should I remove the change of `running-locally.md` here?

**Comment by MrHohn** [comment]
Removed the doc change.

**Pull request merged**